cli/internal/cloudcmd/iam.go

```diff
@@ -95,7 +95,7 @@ type GCPIAMConfig struct {
 
 // AzureIAMConfig holds the necessary values for Azure IAM configuration.
 type AzureIAMConfig struct {
-	Region           string
+	Location         string
```

Comment:
I think the reason we had this named as `Region` before was to have it in line with AWS and GCP, both of which use region instead of Location.
Not sure if we want to keep this though, especially when its already called `Location` in the config.

Reply:
Seems reasonable to name everything region then